scripts/import_macos_pkg_resources.py

```diff
@@ -0,0 +1,127 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This script must be executed at the root of the repository.
+
+import os
+import xml.etree.ElementTree as ET
+
+# List of translation keys
+keys = [ "macosinstaller.title",
+         "macosinstaller.unsupported_version.title",
+         "macosinstaller.unsupported_version.message",
+         "macosinstaller.previous_build.title",
+         "macosinstaller.previous_build.message",
+         "macosinstaller.welcome.message1",
+         "macosinstaller.welcome.message2",
+         "macosinstaller.conclusion.title",
+         "macosinstaller.conclusion.message1",
+         "macosinstaller.conclusion.message2",
+         "macosinstaller.conclusion.message3" ];
+
+# Simple method that retrieves the text value of a localized string.
+def getTranslation(root, string, nodeName):
+    for node in root.findall(f"./context/message[@id='{string}']"):
+        for child in node:
+            if child.tag != nodeName:
+                continue
+            if child.text == None:
+                return ''
+            return child.text
+    return ''
+
+# Generate the translation folder for a particular locale. `nodeTag` can be
+# `source` (for english only) or `translation` (for anything else)
+def translate(root, locale, nodeTag):
+  folder = os.path.join('macos', 'pkg', 'Resources', locale + '.lproj')
+  os.mkdir(folder)
+
+  localizableFile = os.path.join(folder, 'Localizable.strings')
+  f = open(localizableFile, 'w')
+  for key in keys:
+      f.write(f"'{key.replace('macosinstaller.', '').replace('.', '_')}' = '{getTranslation(root, key, nodeTag)}';\n")
```

Comment:
Looking at https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingResources/Strings/Strings.html

They seem to use double quotes as delimiter? Independently from that, I assume you need to escape the delimiter character in the translation.

I also think it would be more readable if you transform the key in a separate variable.